### PR TITLE
Match backend org upload API. Remove watch variable to close modal.

### PR DESCRIPTION
**Pull request by cvializ**
@andrewconner @adamjgrant

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/common/modal/modal.js [hunk 1]
```diff
@@ -57,10 +56,8 @@ angular.module('kifi')
         scope.backdropStyle.opacity = attrs.kfOpacity || 0.3;
         scope.backdropStyle.backgroundColor = attrs.kfBackdropColor || 'rgba(0, 40, 90, 1)';
 
-        scope.$watch('forceClose', function (newVal, oldVal) {
```
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/common/modal/modal.js:60]: this was gross

File: kifi-backend/modules/shoebox/angular/src/common/modal/photoUpload/profileImage.js [hunk 2]
```diff
@@ -241,32 +243,51 @@ angular.module('kifi')
...
+            picHeight: positioning.imageHeight,
+            cropX: Math.floor(scaling * (maskOffset - positioning.currentLeft)),
+            cropY: Math.floor(scaling * (maskOffset - positioning.currentTop)),
+            cropSize: Math.floor(scaling * maskSize)
+          };
+
+          if (scope.uploadUrl && scope.uploadUrl.indexOf('organization') !== -1) {
+            upload = orgProfileService.uploadOrgAvatar(scope.profile.id, data.cropX, data.cropY, data.picWidth, data.picHeight, scope.files[0]);
+            upload.then(function () {
+              scope.$broadcast('forceCloseModal');
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/common/modal/photoUpload/profileImage.js:265]: Should we broadcast on the modal's scope?

Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/common/modal/photoUpload/profileImage.js:265]: Isn't it already the modal's scope? `modal.open` takes the current scope as an argument, so it was my understanding that it somehow inherits from the modal-parent's scope

Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/common/modal/photoUpload/profileImage.js:265]: You're probably right.

**Comment by adamjgrant** [comment]
Looks okay. Sounds like you and @andrewconner are discussing some changes?

**Pull request merged**